[LibWebRTC][WPE] Build fix for Ubuntu 20.04 after 271173@main

dpino opened the pull request:
#### 995816c95079a9d59ea12aae607b6eb9b5a357a8
<pre>
[LibWebRTC][WPE] Build fix for Ubuntu 20.04 after 271173@main
<a href="https://bugs.webkit.org/show_bug.cgi?id=265398">https://bugs.webkit.org/show_bug.cgi?id=265398</a>

Reviewed by Alex Christensen.

* Source/WebKit/Shared/RTCNetwork.serialization.in: Add guard PLATFORM
  is Cocoa.

Canonical link: <a href="https://commits.webkit.org/272602@main">https://commits.webkit.org/272602@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f8f9a8122477bd8b45adfadd278f5d4305192947

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/32218 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/10952 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/34033 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/34778 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/29197 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/13336 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/8162 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/28777 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/32636 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/9263 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/28806 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/8050 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/8216 "Passed tests") | | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/36123 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/29324 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/29173 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/34325 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/8318 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/6257 "Passed tests") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/32188 "Found 1 new API test failure: /WebKitGTK/TestWebKitAccessibility:/webkit/WebKitAccessibility/accessible/state (failure)") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/9959 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/7536 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/8950 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/8857 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

achristensen07 commented on Source/WebKit/Shared/RTCNetwork.serialization.in:
```diff
@@ -55,6 +55,7 @@ struct WebKit::RTCNetwork {
     Vector<WebKit::RTC::Network::InterfaceAddress> ips;
 };
 
+#if PLATFORM(COCOA) || PLATFORM(IOS_FAMILY)
```
This is redundant.  It should be just PLATFORM(COCOA) which includes iOS family OSes.

dpino committed:
[LibWebRTC][WPE] Build fix for Ubuntu 20.04 after 271173@main
https://bugs.webkit.org/show_bug.cgi?id=265398

Reviewed by Alex Christensen.

* Source/WebKit/Shared/RTCNetwork.serialization.in: Add guard PLATFORM
  is Cocoa.

Canonical link: https://commits.webkit.org/272602@main
